repository_service_tuf_api/config.py

```diff
@@ -43,13 +44,16 @@ class Config:
             content = f.read()
         example_settings = json.loads(content)
 
-        schema_extra = {
-            "example": {
-                "message": "Settings successfully submitted.",
+        example = {
+            "data": {
                 "task_id": "06ee6db3cbab4b26be505352c2f2e2c3",
-            }
+                "last_update": "2022-12-01T12:10:00.578086",
```

Comment:
I see you added `last_update` as an example, but you missed adding it in the `PutResponse` class as an attribute.

You will also need to update the `test_config` tests which are testing the `PUT /api/v1/config` API endpoint.